
2014/11/05 16:43:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "js/jquery.min.map".' in C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\OpenServer\domains\ybugs.local\framework\web\CWebApplication.php(141): CWebApplication->runController('js/jquery.min.m...')
#1 C:\OpenServer\domains\ybugs.local\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\OpenServer\domains\ybugs.local\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/js/jquery.min.map
HTTP_REFERER=http://ybugs.local/en
---
2014/11/06 16:47:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php:286
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2014/11/06 16:59:18 [error] [exception.CException] exception 'CException' with message 'IssuesController cannot find the requested view "get_iisue".' in /Applications/AMPPS/www/ybugs.local/framework/web/CController.php:878
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(782): CController->renderPartial('get_iisue', Array, true)
#1 /Applications/AMPPS/www/ybugs.local/protected/controllers/IssuesController.php(38): CController->render('get_iisue', Array)
#2 [internal function]: IssuesController->actionGetissue('1')
#3 /Applications/AMPPS/www/ybugs.local/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs(Object(IssuesController), Array)
#4 /Applications/AMPPS/www/ybugs.local/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(IssuesController), Object(ReflectionMethod), Array)
#5 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(282): CController->run('getissue')
#9 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('issues/getissue')
#10 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#11 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/en/issues/getissue/1
HTTP_REFERER=http://ybugs.local/en/issues/list/1
---
2014/11/06 18:50:58 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CDbCommand.queryAll".' in /Applications/AMPPS/www/ybugs.local/framework/base/CComponent.php:130
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/protected/models/ext/ExtIssues.php(30): CComponent->__get('queryAll')
#1 /Applications/AMPPS/www/ybugs.local/protected/controllers/IssuesController.php(36): ExtIssues->getIssue('3')
#2 [internal function]: IssuesController->actionGetissue('3')
#3 /Applications/AMPPS/www/ybugs.local/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs(Object(IssuesController), Array)
#4 /Applications/AMPPS/www/ybugs.local/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(IssuesController), Object(ReflectionMethod), Array)
#5 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#6 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#7 /Applications/AMPPS/www/ybugs.local/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(282): CController->run('getissue')
#9 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('issues/getissue')
#10 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#11 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/ru/issues/getissue/3
HTTP_REFERER=http://ybugs.local/ru/issues/list/2
---
2014/11/06 19:42:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "issue/getissue".' in /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php:286
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('issue/getissue')
#1 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/ru/issue/getissue/
HTTP_REFERER=http://ybugs.local/ru/issues/list/2
---
2014/11/06 19:42:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "issue/getissue".' in /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php:286
Stack trace:
#0 /Applications/AMPPS/www/ybugs.local/framework/web/CWebApplication.php(141): CWebApplication->runController('issue/getissue')
#1 /Applications/AMPPS/www/ybugs.local/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /Applications/AMPPS/www/ybugs.local/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/ru/issue/getissue/3
HTTP_REFERER=http://ybugs.local/ru/issues/list/2
---
